backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/Migrations.java

```diff
@@ -127,6 +127,9 @@ public static synchronized void initOrMigrate(SqlDatabase db) {
             executeMigrationFile(
                 tdb, "migration21.sql", "add exist and range role to schemas and metadata");
 
+          if (version < 22)
+            executeMigrationFile(tdb, "migration22.sql", "add script failure address");
```

Comment:
edit to say: add script failure email address

Reply:
This comment is mute if you move it to other class

Reply:
Moved to the ScriptService